core/invoker/src/main/scala/whisk/core/invoker/InvokerReactive.scala

```diff
@@ -83,7 +85,7 @@ class InvokerReactive(config: WhiskConfig, instance: InstanceId, producer: Messa
           "--cap-drop" -> Set("NET_RAW", "NET_ADMIN"),
           "--ulimit" -> Set("nofile=1024:1024"),
           "--pids-limit" -> Set("1024"),
-          "--dns" -> config.invokerContainerDns.toSet) ++ logsProvider.containerParameters)
+          "--dns" -> containerArgs.dnsServers.toSet) ++ logsProvider.containerParameters)
```

Comment:
I'm wondering: The dns configuration seems duplicated (given it's set in the factory itself again as well). Can we remove it here and thus remove the dependency on the config from `InvokerReactive`?

Reply:
well, now I'm noticing a problem that _transformEnvironment.sh_ is ignoring my env vars that end with index, e.g. `CONFIG_whisk_containerFactory_containerArgs_dnsServers.0=8.8.8.8`; will look into that...

Reply:
oh, using `_<index>` works...